services/explorer/src/pages/Home/ChartContainer.tsx

```diff
@@ -81,44 +71,55 @@ const ChartContainer = ({
             },
           ]}
         />
-        {/* <BarChart
+        <BarChart
           title='DAG Block Per Second'
           labels={dagBlocks
-            .slice(5)
-            .map((block) => block.pbftPeriod.toString())}
+            .slice(6)
+            .reverse()
+            .map((block) => block.level.toString())}
           datasets={[
             {
-              data: dagBlocks?.slice(5).map((block) => {
-                return (
-                  block.timestamp -
-                  dagBlocks.find((b) => b.pbftPeriod === block.pbftPeriod - 1)
-                    .timestamp
-                );
-              }),
+              data: dagBlocks
+                ?.slice(0, 5)
+                .reverse()
+                .map(
+                  (block) =>
+                    block.transactionCount /
+                    (block.timestamp -
+                      dagBlocks.find((b) => b.level === block.level - 1)
+                        .timestamp) /
+                    1000
+                ),
               borderRadius: 5,
               barThickness: 20,
               backgroundColor: theme.palette.secondary.main,
             },
           ]}
         />
-        {/* <BarChart
+        <BarChart
           title='DAG efficiency'
           labels={dagBlocks
-            .sort((a, b) => +b.timestamp - +a.timestamp)
-            .slice(0, 6)
-            .flatMap((x) => x.level)}
+            .slice(6)
```

Comment:
I think we can leave the last 10 instead of 6 to be consistent with the two cards. @VargaElod23  what do you think?

Reply:
I agree. or even make it dynamic and set it to 10 by default